.github/workflows/pytest.yml

```diff
@@ -0,0 +1,33 @@
+name: pytest
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  pytest:
+    name: "pytest"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: "CentOS Stream 8"
+            image: "quay.io/centos/centos:stream8"
+            pytest_args: ''
+
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ matrix.image }}
+
+    steps:
+      - name: "Checkout repository"
+        uses: actions/checkout@v3
+
+      - name: "Run container-pre-test.sh"
+        run: |
+          bash scripts/container-pre-test.sh
+
+      - name: "Run nosetest"
+        run: |
+          nosetests -v -c nosetest.ini
```

Comment:
as mentioned, IMHO would make sense to use the existing configuration, instead of adding a 3rd one (or 4th if we count also the suse-specific one)